### PR TITLE
Overhaul of machine issues command.

### PR DESCRIPTION
**Pull request by Gerrit91**
- Can now show multiple issues per machine
- Dedicated table printer for issues
- Added test for overlapping ASN

### CODE REVIEW

File: cmd/printer.go [hunk 1]
```diff
@@ -677,6 +684,38 @@ func (m MetalMachineTablePrinter) Print(data []*models.V1MachineResponse) {
...
+		if machine.Partition != nil && machine.Partition.ID != nil {
+			partition = *machine.Partition.ID
+		}
+		project := ""
+		if machine.Allocation != nil && machine.Allocation.Project != nil {
+			project = *machine.Allocation.Project
+		}
+
+		issues := ""
+		for _, issue := range is {
```
Comment (majst01) [inline cmd/printer.go:708]: better with strings.Join ?

Comment (Gerrit91) [reply cmd/printer.go:708]: But how to prepend each issue with a hyphen? Somehow this code does not even work. Either with join nor with the loop I see any line breaks. :(

**Comment by majst01** [comment]
looks better now, albeit NAME and PROJECT column are empty

**Comment by Gerrit91** [comment]
Good catch. I think, now it should be fine.

**Review by majst01 — APPROVED** [review]

**Pull request merged**